avocado/utils/software_manager.py

```diff
@@ -517,6 +536,91 @@ def provides(self, name):
         else:
             return None
 
+    def info(self, name):
+        """
+        Returns a name for the given package [name]
+
+        :param name: name of the package
+        """
+        cmd = "%s info %s" % (self.base_command, name)
+        full_name = ""
+
+        try:
+            result = process.system_output(cmd)
+            for line in result.splitlines():
+                if line.startswith('Name'):
+                    full_name += line.split()[-1]
+                    full_name += '-'
```

Comment:
why not `full_name += "%s-" % line.split()[-1]`

Reply:
makes sense